gobcore/standalone.py

```diff
@@ -63,7 +63,7 @@ def run_as_standalone(
         params={"stream_contents": False}
     )
     handler = _get_handler(args.handler, service_definition)
-    logger.configure(message_in, get_logger_name(handler))
+    logger.configure(message_in, get_logger_name(handler), standalone=True)
```

Comment:
Can we solve this differently? Logger should not "know" about things as "standalone" or not. Maybe let the calling function (run_as_standalone) pass the StdOutHandler? The requesting function "knows" where it wants the logging to end up. The logger itself should be oblivious to this.